README.md

```diff
@@ -158,6 +178,54 @@ These are the VS Code [settings] available for the Extension:
 [workspace level]:
   https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings
 
+## Developer roadmap
```

Comment:
Let's actually create a board and put a link to it instead? For now either tickets or stubs are fine, we'll review it later

Reply:
For now I made it into a lighter-looking bullet list since I can't make the board myself. Cc @mattseddon happy to link instead once one exists.

See https://github.com/iterative/vscode-dvc/tree/readme/known-limitations#developer-roadmap. Thanks

Reply:
We'll go through Q planning next week.

Reply:
So should we keep this open or merge (since it's approved) and revisit then?

Reply:
If the PR meets the criteria to merge then feel free. We should keep everything moving.